system-parachains/asset-hubs/asset-hub-kusama/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -47,321 +47,331 @@ use core::marker::PhantomData;
 
 /// Weight functions for `pallet_xcm_benchmarks::generic`.
 pub struct WeightInfo<T>(PhantomData<T>);
-impl<T: frame_system::Config> WeightInfo<T> {
+impl<T: frame_system::Config> pallet_xcm_benchmarks::generic::WeightInfo for WeightInfo<T> {
```

Comment:
@karolk91 there is no template used for xcm weights here in fellows, so you need to manually revert back this for all:
```suggestion
impl<T: frame_system::Config> WeightInfo<T> {
```
also you need to revert all those `fn` -> `pub(crate) fn`

I don't remember the reason why the template was not added here, maybe we should add it, so please fix or create separate issue/PR to add one

Reply:
https://github.com/polkadot-fellows/runtimes/issues/726

Reply:
thanks @bkontur I will work on this but will start no earlier than 20-21 May

Reply:
@karolk91 xcm template is merged and works, which is cool, and the fix for `pay_fees` is also merged, so please, update this branch on actual `main` and re-run this command: https://github.com/polkadot-fellows/runtimes/pull/709#issuecomment-2851080995

so we can continue with this PR

Reply:
@karolk91 there are compilation issues for xcm benchmarks, because the bot didn't have that template, please, fix those manually and then I will re-run